pyproject.toml

```diff
@@ -0,0 +1,199 @@
+# https://packaging.python.org/en/latest/specifications/pyproject-toml/
+[build-system]
+requires = ["pdm-backend", "wheel>=0.42"]
+build-backend = "pdm.backend"
+
+[project]
+name = "pymediainfo"
+description = "A Python wrapper for the mediainfo library."
+authors = [
+    {name = "Louis Sautier", email = "sautier.louis@gmail.com"},
+]
+readme = "README.rst"
+license = {text = "MIT"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+]
+dynamic = ["version"]
+requires-python = ">=3.9"
+dependencies = [
+]
+
+[project.optional-dependencies]
+tests = [
+    "pytest>=6",
+    "pytest-cov",
+    "pytest-xdist",
+    "coverage[toml]>=7.2",
+]
+docs = [
```

Comment:
Do you know if there is a way to not duplicate the definition of these dependencies between tox.ini and this file?

Reply:
Yes, with the `extras` key:
```
[testenv]
extras = tests,dev
```
But I found that it doesn't work sometimes, but I' m not very familiar with tox.

Reply:
OK https://tox.wiki/en/latest/config.html#extras really sounds nice. Do you remember what issues you faced with it? I like not duplicating information (like the version which is obtained from the SCM).

Reply:
I had issues when using a command with `skip_install=false` (which makes sense now that I'm thinking about it 😄). `docs` is sometimes used with `skip_install=false` for instance, but not here so it should be fine.
If you know about tox you should experiment. 
I was actually planning another PR to get rid of tox and use pdm scripts instead 🙃

Reply:
>  I was actually planning another PR to get rid of tox and use pdm scripts instead 🙃

OK I won't spend any more time on tox for now then!